end-to-end-use-cases/coding/text2sql/README.md

```diff
@@ -1,30 +1,24 @@
-## Text2SQL: Natural Language to SQL Interface
+# Text2SQL: Evaluating and Fine-tuning Llama Models
 
-This project provides a set of scripts to convert natural language queries into SQL statements using Meta's Llama model. The goal is to enable users to interact with databases using natural language inputs, making it easier for non-technical users to access and analyze data. 
+This folder contains scripts to:
 
-For detailed instructions on setting up the environment, creating a database, and executing natural language queries using the Text2SQL interface, please refer to the quickstart.ipynb notebook.
+1. Evaluate Llama (original and fine-tuned) models on the Text2SQL task using the popular [BIRD](https://bird-bench.github.io) dataset in **3 simple steps**;
 
-### Structure:
+2. Generate fine-tuning datasets (both with and without CoT reasoning) and fine-tuning Llama 3.1 8B with the datasets, gaining a **165% (with no reasoning) and 209% (with reasoning) accuracy improvement** over the original model.
```

Comment:
Have an intro on the main method, how did we get better data, we added COT using synth-kit? then numbers we got x% improvement over the baseline.  

Can we please also clarify on generating data with non-COT? IF COT is giving the best result whats the reason for keeping non-COT data? there might be some value in showing how COT can beat none COT generated data which needs some language tuning.

Reply:
OK will add some intro on the main method, the baseline number, and comparison of the CoT and non CoT dataset. No I didn't use synthetic kit because I found it harder to use than writing the script fine_tuning/create_reasoning_dataset.py from scratch. @init27 may think different - then please use this as a good example of showing the value of the data kit.